themes/console-home/static/js/page-type/feature.js

```diff
@@ -356,14 +361,26 @@ class Filter {
         if (totalActiveFilters == totalFilters) this.form.classList.add("is-showing-all");
     }
 
-    updateInfo() {
+    updateInfo(info) {
+        if (info) {
+            if (info.type == "search") {
+                this.searchTerm = info.content;
+            }
+        }
+        let searchFeedback = this.searchTerm != "" ? 'Searching "' + this.searchTerm + '"' : "";
```

Comment:
This has an XSS vulnerability whereby you can inject code into the page e.g. enter this:

`"><a href="https://example.com">Example</a>`

And it'll be output to the page as a link. Can you sanitize the input?

Reply:
Right, done, let me know if that's sufficient.